src/botnet.c

```diff
@@ -169,9 +164,14 @@ int addparty(char *bot, char *nick, int chan, char flag, int sock,
     }
   }
   /* New member */
-  if (parties == maxparty) {
-    maxparty += 50;
-    party = (party_t *) nrealloc((void *) party, maxparty * sizeof(party_t));
+  if (!party_size) {
+      party_size = 1;
+      party = nmalloc(party_size * sizeof(party_t));
+  }
+  else if (parties == party_size) {
+    party_size <<= 1;
```

Comment:
*=2 compiles to the same instruction and is less confusing for humans to read but ok :)

Reply:
yes and yes, old habit, next time ill change or remove something like this i may remember and change all such occurrences.